webapp/niapp.yaml

```diff
@@ -416,3 +442,21 @@ definitions:
     format: binary
     title: WebApp Content
     description: The webapp binary content. Depending on the webapp's type it can be a dashboard, template or *.nipkg file exported from LabVIEW NXG
+  WebAppsAdvancedQuery:
+    type: object
+    title: Advanced Query Object for WebApps
+    properties:
+      filter:
+        description: The webapp query filter in Dynamic Linq
+        type: string
+        example: name.Contains("myWebApp") || type == "WebVI"
```

Comment:
Maybe we should list all the supported fields and operators. Otherwise it is trial and error for the users

Reply:
Done. I based the format of the documentation on the APM swagger doc.